### PR TITLE
Feature change login process

### PR DESCRIPTION
**Pull request by eduardomiyoshizup**

### CODE REVIEW

File: moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt [hunk 1]
```diff
@@ -107,6 +108,14 @@ class MooveExceptionHandler(private val messageSource: MessageSource) {
                 ?: ex.message)
     }
 
+    @ExceptionHandler(UnauthorizedException::class)
```
Comment (emiteze) [inline moove/api/src/main/kotlin/io/charlescd/moove/api/MooveExceptionHandler.kt:111]: Maybe we should change from UnauthorizedException to ForbiddenException, cause unauthorized reminds of 401, which is not correct for our situation.

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt [hunk 2]
```diff
@@ -56,5 +56,6 @@ enum class MooveErrorCode(val key: String) {
     MODULE_MUST_HAVE_AT_LEAST_ONE_COMPONENT("module.must.have.at.least.one.component"),
     CREATE_USER_ERROR_EMAIL_ALREADY_EXISTS("create.user.error.email.already.exists"),
     USER_PASSWORD_DOES_NOT_MATCH("user.password.does.not.match"),
-    CANNOT_RESET_YOUR_OWN_PASSWORD("cannot.reset.your.own.password")
+    CANNOT_RESET_YOUR_OWN_PASSWORD("cannot.reset.your.own.password"),
+    NOT_AUTHORIZED("not.authorized")
```
Comment (emiteze) [inline moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt:60]: Also we should change this error code to FORBIDDEN instead

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/exceptions/UnauthorizedException.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package io.charlescd.moove.domain.exceptions
+
+class UnauthorizedException : RuntimeException("Access denied!")
```
Comment (emiteze) [inline moove/domain/src/main/kotlin/io/charlescd/moove/domain/exceptions/UnauthorizedException.kt:21]: Maybe we should change from UnauthorizedException to ForbiddenException, cause unauthorized reminds of 401, which is not correct for our situation.

File: moove/web/src/main/resources/locale/messages.properties [hunk 4]
```diff
@@ -27,3 +27,4 @@ user.already.disassociated=This user is already disassociated from this user gro
 create.user.error.email.already.exists=The email {0} is already registered.
 user.password.does.not.match=The password informed does not match.
 cannot.reset.your.own.password=Cannot reset your own password.
+not.authorized=Not Authorized
```
Comment (emiteze) [inline moove/web/src/main/resources/locale/messages.properties:30]: Also change this message to a forbidden message

File: moove/web/src/main/resources/locale/messages_pt_BR.properties [hunk 5]
```diff
@@ -26,3 +26,4 @@ user.already.disassociated=Este usuário ja esta desassociado deste grupo.
 create.user.error.email.already.exists=O email {0} já está registrado.
 user.password.does.not.match=A senha informada está incorreta.
 cannot.reset.your.own.password=Não é permitido resetar a própria senha.
+not.authorized=Sem autorização
```
Comment (emiteze) [inline moove/web/src/main/resources/locale/messages_pt_BR.properties:29]: Also change this message to a forbidden message

**Review by emiteze — CHANGES_REQUESTED** [review]

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt [hunk 6]
```diff
@@ -57,5 +57,5 @@ enum class MooveErrorCode(val key: String) {
     CREATE_USER_ERROR_EMAIL_ALREADY_EXISTS("create.user.error.email.already.exists"),
     USER_PASSWORD_DOES_NOT_MATCH("user.password.does.not.match"),
     CANNOT_RESET_YOUR_OWN_PASSWORD("cannot.reset.your.own.password"),
-    NOT_AUTHORIZED("not.authorized")
+    forbidden("forbidden")
```
Comment (emiteze) [inline moove/domain/src/main/kotlin/io/charlescd/moove/domain/MooveErrorCode.kt:60]: Put this ErrorCode in upper case

File: moove/web/src/main/resources/locale/messages_pt_BR.properties [hunk 7]
```diff
@@ -26,4 +26,4 @@ user.already.disassociated=Este usuário ja esta desassociado deste grupo.
 create.user.error.email.already.exists=O email {0} já está registrado.
 user.password.does.not.match=A senha informada está incorreta.
 cannot.reset.your.own.password=Não é permitido resetar a própria senha.
-not.authorized=Sem autorização
+forbidden=Forbidden
```
Comment (emiteze) [inline moove/web/src/main/resources/locale/messages_pt_BR.properties:29]: Translate this message to portuguese

**Review by emiteze — CHANGES_REQUESTED** [review]

**Review by emiteze — APPROVED** [review]

**Review by emiteze — DISMISSED** [review]

**Review by pedroborgeszup — APPROVED** [review]

**Review by ifdouglas — APPROVED** [review]

**Review by emiteze — APPROVED** [review]

**Pull request merged**